### DISCUSSION TITLE
can you add a drag and drop for copying?

### DISCUSSION THREAD

**Discussion by alexander-aleksandrov** [OliBlade/3Cols#33] [Ideas]
Thanks a lot for your work! It is actualy great. Right now your product is the most simple to use and based on simple ideas. 
I've met one problem that I instinctively want to drag-n-drop subcategory to another category. If you'll have a time to make this happen this would be great since then we can reorganize the structure as usual in the file system. 

Thank in advance

**Comment by OliBlade** [comment]
Glad to hear you're enjoying 3Cols! 

This isn't the first time this idea has come up, and I like the idea. I've created an issue for this for future review [here](https://github.com/OliBlade/3Cols/issues/34).
